pkg/stores_test.go

```diff
@@ -0,0 +1,68 @@
+package ocr2keepers
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+type checkResult struct {
+	Retryable bool
+	Data      string
+}
+
+func TestNewResultStore(t *testing.T) {
```

Comment:
i personally find table driven tests much easier to read and to see what things are being tested. You don't need to change it since we haven't really agreed to this as a team. But maybe if you like the idea we can work on making new tests follow the pattern.
in GoLand you can have it generate the shell for you it looks like this
```
func Test_resultStore_Add(t *testing.T) {
	type args[T any] struct {
		result T
	}
	type testCase[T any] struct {
		name string
		s    resultStore[T]
		args args[T]
	}
	tests := []testCase[ /* TODO: Insert concrete types here */ ]{
		// TODO: Add test cases.
	}
	for _, tt := range tests {
		t.Run(tt.name, func(t *testing.T) {
			tt.s.Add(tt.args.result)
		})
	}
}
```

Reply:
Oh yeah me too! I just didn't use them this time because the functionality here is quite limited and linear, like it either adds or it doesn't and it either removes or it doesn't, so I thought a test table would get messy? I usually use test tables for testing functions that return values I guess?